src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1590,6 +1590,44 @@ protected override void HandleSetBreakpointsRequestAsync(IRequestResponder<SetBr
                     var resBreakpoints = new List<Breakpoint>();
                     foreach (var bp in breakpoints)
                     {
+                        if (dict.ContainsKey(bp.Line))
+                        {
+                            // already created
+                            IDebugBreakpointRequest2 breakpointRequest;
+                            if (dict[bp.Line].GetBreakpointRequest(out breakpointRequest) == 0 && 
+                                breakpointRequest is AD7BreakPointRequest ad7BPRequest)
+                            {
+                                // Check to see if this breakpoint has a condition that has changed.
+                                if (!StringComparer.InvariantCulture.Equals(ad7BPRequest.Condition, bp.Condition))
```

Comment:
nit: Is there a reason you are using `InvariantCulture` instead of an ordinal string compare?